src/templates/Studio/Wonder/Sun/Sun.js

```diff
@@ -1,9 +1,5 @@
-import React, { useEffect } from 'react';
-import {
-  Animation,
-  DirectionalLight,
-  Color3,
-} from 'babylonjs';
+import { useEffect } from 'react';
```

Comment:
Is the lighting of the sculpture still based on the position of the sun in Amsterdam or is this legacy code that's not really used anymore?

Reply:
Still using that beautiful code written by Andy, couldn't lift my hand to delete it